internal/pkg/doc/doc_test.go

```diff
@@ -243,6 +243,88 @@ func TestInputsFromVariablesTf(t *testing.T) {
 	assert.Equal(t, expected, actual)
 }
 
+func TestInputsSortedByRequired(t *testing.T) {
```

Comment:
Great you've added a test for the `doc.SortInputsByRequired` method. May I ask you to create the missing test for `doc.SortInputsByName`?